chatkit-core/src/main/kotlin/com/pusher/chatkit/CurrentUser.kt

```diff
@@ -19,6 +19,27 @@ import elements.Subscription
 class CurrentUser(
         private val syncCurrentUser: SynchronousCurrentUser
 ) {
+
+    //Delegate fields for the underlying SynchronousCurrentUsers
+    val id: String
+        get() = syncCurrentUser.id
+
+    var name: String?
+        get() = syncCurrentUser.name
+        set(name) { syncCurrentUser.name = name}
```

Comment:
I don't think they are settable, are they? They shouldn't be.

Reply:
right - saw them as vars and made them settable. makes sense to not make them in this context